proposal/WarClans612/proposal.rst

```diff
@@ -0,0 +1,171 @@
+===========================
+Proposal for [Automatic Tag Generation for Text Article]
+===========================
+
+Basic information
+=================
+
+`The GitHub repository <https://github.com/WarClans612/Automatic-Tag-Generation-for-Text-Article>`__
+
+Text content is unstructured and complex information that is challenging to search and index,
+therefore, by using tag information accompanied with the text content, we could extract main idea
+and topic within the unstructured information.
+
+Problem to solve
+================
+
+Industry Problem
+-----------------
+
+Nowadays, thanks to the development of social media and internet, more information can be easily
+accessible from anywhere. However, more problem arises as information flow faster, which overwhelm
+manual labor for many tasks. Most of the information in the web is in the shape if text content that
+by itself is hard to be digested by computer due to misaligned language used.
+
+Social media, content provider, modern encyclopedia (e.g. Wikipedia) consists of many content which 
+can be grouped into bags of topic. Using manual labor to separate and tag each piece of pages usually 
+yield the best results in correctness and precision of the tagged results. This approach cannot keep 
+up with the growth pace of the internet itself, so we would like to make an automatic approach for
+this problem.
+
+Tagging generation can be described simpler as structured information extraction from unstructured
+information (such as text) to enable easier digestion of information and analysis.
+
```

Comment:
It would provide useful information for development if you include the probable size of data to be processed.

Reply:
Added more information in this commit fe9b113